go/vt/vtgate/planbuilder/operators/update.go

```diff
@@ -27,18 +31,27 @@ import (
 	"vitess.io/vitess/go/vt/vtgate/vindexes"
 )
 
-type Update struct {
-	QTable              *QueryTable
-	VTable              *vindexes.Table
-	Assignments         map[string]sqlparser.Expr
-	ChangedVindexValues map[string]*engine.VindexValues
-	OwnedVindexQuery    string
-	AST                 *sqlparser.Update
```

Comment:
Why are we not storing AST in Update anymore, but we are doing so for Delete and Inserts?

Reply:
we want to get rid of using them for others as well, just that subquery is not supported for delete and insert queries therefore it is not changed in this PR.